airbyte-integrations/connectors/source-instagram/acceptance-test-config.yml

```diff
@@ -39,21 +39,35 @@ acceptance_tests:
       - config_path: "secrets/config.json"
         configured_catalog_path: "integration_tests/configured_catalog.json"
         ignored_fields:
-          "user_insights":
-            - email_contacts
-            - follower_count
-            - get_directions_clicks
-            - impressions
-            - phone_call_clicks
-            - profile_views
-            - reach
-            - text_message_clicks
-            - website_clicks
-            - impressions_week
-            - reach_week
-            - impressions_days_28
-            - reach_days_28
-            - online_followers
+          user_insights:
```

Comment:
Which user are we using here?  If this is a fake/test user, this data shouldn't change.  If this is a "real" user, then this information would be changing, but then is inappropriate for this test (not a sandbox)